.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you remove this?

Reply:
Ahhh can we add `.vscode` in gitignore ? :P

Reply:
ofc! please do